nab/detectors/numenta/README.md

```diff
@@ -0,0 +1,119 @@
+# Numenta and NumentaTM detectors
+
+This directory holds the Python 2 code required to run the `numenta` and
+`numentaTM` detectors against the NAB data. In 2019 the main body of the
+benchmark's code was ported to Python 3 however these detectors rely on NuPIC
+which is Python 2 only.
+
+This code can be used to replicate results listed on the scoreboard of
+the main repository for the following detectors:
+
+    numenta
+    numentaTM
+
+## Installation
+
+We assume you have a working version of Python 3 installed as your default Python.
+If your default system Python is still Python 2 you can skip the virtual environment
+creation below.
+
+### Requirements to install
+
+- [Python 2.7](https://www.python.org/download/)
+- [Virtualenv](https://pypi.org/project/virtualenv/)
+
+### Install a virtual environment
+
+Create a new Python 2 virtual environment in this directory.
+
+`virtualenv -p path/to/python2 env`
+
+On Windows this might be:
+
+`virtualenv -p C:\Python27\python.exe env`
+
+Activate that virtual environment.
+
+`./env/Scripts/activate`
+
+or
+
+`env\Scripts\activate.bat` on Windows.
+
+Confirm you have a local Python 2
+
+```
+$ python
+Python 2.7.13 (v2.7.13:a06454b1afa1, Dec 17 2016, 20:53:40) [MSC v.1500 64 bit (AMD64)] on win32
+Type "help", "copyright", "credits" or "license" for more information.
+>>>
+```
+
+### Install detectors
+
+`cd /path/to/NAB/`
+`pip install nupic`
```

Comment:
I thought this won't work after the end of this year....?? Weren't we also going to supply a docker image to avoid pip install? I think @lscheinkman has one available.

Reply:
`pip install nupic` should still work from a Python 2 runtime. I can add the docker image once / if we have it.

Reply:
`pip install nupic` will still work for the foreseeable future. Yes, talk to Luiz about the docker image.

Reply:
Did you resolve the docker image question?